src/cache.py

```diff
@@ -52,6 +79,8 @@ def wrapper(*args, **kwargs):  # noqa
             if var in route_variables:
```

Comment:
Just checking as you mentioned this briefly @peterdudfield, is there anything else you can see in this dict that could be removed to make the keys more reusable/less long? Other than `["session", "user", "request"]`

Reply:
Thanks, there might be, I'll put them in a different issue as we might be able to get rid of some

Reply:
https://github.com/openclimatefix/uk-pv-national-gsp-api/issues/319